app/controllers/topbar_sweeper.rb

```diff
@@ -13,15 +13,15 @@ def after_destroy(record)
     record.expire_topbar_cache(self)
   end
 
-  def after_select(record)
-    record.expire_topbar_cache(self)
-  end
-
```

Comment:
I can't imagine why would we invalidate cache on each select. It does not seem to be valid hook at all as well.